app/views/shared/_supporting_information.html.erb

```diff
@@ -115,6 +112,38 @@
       <p class='govuk-body'>Consultation is not applicable for proposed permitted development.</p>
     </div>
   </div>
+  <div class="govuk-accordion__section">
+    <div class="govuk-accordion__section-header">
+      <h2 class="govuk-accordion__section-heading">
+        <span class="govuk-accordion__section-button" id="audit-log">
+          Audit log
+        </span>
+        <span class="govuk-accordion__icon" aria-hidden="true"></span>
+      </h2>
+    </div>
+    <div class="govuk-accordion__section-content" aria-labelledby="audit-log">
+      <% if @audit.present? %>
+        <p class="govuk-body">
+          <% if @audit.activity_type.match("/*_validation_request_cancelled") %>
+            <%= render "audits/types/validation_request_cancelled", locals: { item: @audit } %>
+          <% elsif @audit.activity_type.include?("request") || @audit.activity_type.include?("document_received_at_changed") %>
+            <%= render "audits/types/#{@audit.activity_type}", locals: { item: @audit } %>
+          <% else %>
+            <%= render "audits/types/generic_audit_entry", locals: { item: @audit } %>
+          <% end %>
+          <p class="govuk-body">
+            <%= define_audit_user(@audit) %>
+          </p>
+          <p class="govuk-body">
+            <%= @audit.created_at.to_date %> at <%= @audit.created_at.strftime("%H:%M") %> 
```

Comment:
This should be the regular `day_month_date`, it's defined for Time too: https://github.com/unboxed/bops/blob/master/config/initializers/date_formatter.rb#L6

Reply:
but that gives me the whole thing, when I just want the date so then I can match the design to have the created_date at xtime?

so just created at:
![image](https://user-images.githubusercontent.com/9452321/145025864-b357cb4a-1e44-455b-b729-731f943bf383.png)
vs this:
![image](https://user-images.githubusercontent.com/9452321/145025987-7350738b-0db8-42a9-b850-3aed88064fd6.png)

Reply:
For tone consistency I suggest we update the format of the timely one to include the "at". Will look better in our emails too.